CUSTOMISING.md

```diff
@@ -1,52 +1,50 @@
-
 # Customisation
 
 ## Customizing DAMAP backend
 
 To migrate with the base, you need to fetch updates with upstream, following Semantic Versioning guidelines:
+
 - **Major:** Big changes (first value).
 - **Minor:** Additional changes (second value).
-For more details on Semantic Versioning, see [semver.org](https://semver.org/).
+  For more details on Semantic Versioning, see [semver.org](https://semver.org/).
 
-In the `pom.xml`, update the dependency version as needed to match the new release of the base backend, for example:
+### Dependency Management
 
-```xml
-  <!-- Base image -->
-  <dependency>
-    <groupId>org.damap</groupId>
-    <artifactId>base</artifactId>
-    <version>4.0.2</version>
-  </dependency>
-```
+There are two options for handling the DAMAP dependency:
 
-You can implement your institutional backend by creating a new project with [quarkus](https://quarkus.io/).
+1. **Maven Central Approach**: You can specify the DAMAP version as a dependency, and Maven will automatically pull it from [Maven Central](https://central.sonatype.com/artifact/org.damap/base).
 
-Download the damap-backend project and build it using:
+   Add the following dependency to your `pom.xml`:
 
-```shell
-mvn clean install -Dquarkus-profile=dev
-```
+   ```xml
+   <dependency>
+     <groupId>org.damap</groupId>
+     <artifactId>base</artifactId>
+     <version>4.0.2</version>
+   </dependency>
+   ```
 
-After compiling the backend, you'll have a jar file in the target folder (e.g., `damap-backend-1.2.0.jar`).
+2. **Local Build Approach**: Alternatively, you can download the DAMAP Git repository, compile it locally, and specify the compiled version as a dependency. When doing this, Maven will prioritize the locally compiled version over the one from Maven Central.
 
-To include this in your institutional project, add the following dependency to your `pom.xml`:
+   To compile locally, run the following:
 
-```xml
-<dependency>
-  <groupId>org.damap</groupId>
-  <artifactId>base</artifactId>
-  <type>jar</type>
-  <version>1.2.0-SNAPSHOT</version>
-</dependency>
-```
+   ```shell
+   git clone https://github.com/tugraz-rdm/damap-backend-tugraz.git
+   cd damap-backend
+   mvn clean install
+   ```
 
-Ensure the jar is either available locally or through your institutional Nexus.
+   Then, add the dependency to your `pom.xml`:
 
-You can run the project using:
+   ```xml
+   <dependency>
+     <groupId>org.damap</groupId>
+     <artifactId>base</artifactId>
+     <version>1.2.0-SNAPSHOT</version>

```

Comment:
comment: Using the same version as above could minimize confusion. However you prefer.